source/wp-content/themes/wporg-news-2021/block-templates/index.html

```diff
@@ -1,31 +1,23 @@
 <!-- wp:wporg/global-header /-->
 
-<!-- wp:query {"tagName":"main","queryId":1,"query":{"perPage":10,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","sticky":""}} -->
-<main class="wp-block-query">
-<!-- wp:post-template -->
-<!-- wp:group {"layout":{"inherit":true}} -->
-<div class="wp-block-group">
-	<!-- wp:post-title {"isLink":true} /-->
-	<!-- wp:post-featured-image {"isLink":true} /-->
-	<!-- wp:post-excerpt /-->
-	<!-- wp:spacer {"height":40} -->
-	<div style="height:40px" aria-hidden="true" class="wp-block-spacer"></div>
-	<!-- /wp:spacer -->
-</div>
+<!-- wp:group {"tagName":"header","className":"header-archive header-archive-all-posts","layout":{"inherit":true}} -->
+<header class="wp-block-group header-archive header-archive-all-posts">
+	<!-- wp:heading {"level":1,"align":"wide"} -->
+	<h1 class="alignwide">All Posts</h1>
+	<!-- /wp:heading -->
+</header>
 <!-- /wp:group -->
-<!-- /wp:post-template -->
-<!-- wp:group {"layout":{"inherit":true}} -->
-	<div class="wp-block-group">
-	<!-- wp:query-pagination -->
-	<div class="wp-block-query-pagination"><!-- wp:query-pagination-previous /-->
 
-		<!-- wp:query-pagination-numbers /-->
+<!-- wp:query {"tagName":"main","className":"site-content-container content-archive"} -->
+<main class="wp-block-query site-content-container content-archive">
+	<!-- wp:post-template -->
+		<!-- wp:template-part {"slug":"content-archive","theme":"wporg-news-2021","tagName":"article","layout":{"inherit":true}} /-->
```

Comment:
Is `theme` necessary? I was thinking it defaulted to the current one.

Reply:
🤔 Good question. I have just been including it because Blockbase does. I'll test without it and report back...

Reply:
Removed: 3339333